packages/web/src/scss/components/SplitButton/index.html

```diff
@@ -0,0 +1,671 @@
+{{#> web/layout/default title="SplitButton" parentPageName="Components" }}
+<section class="UNSTABLE_Section">
+  <div class="Container">
+    <h2 class="docs-Heading">Primary</h2>
+
+    <div class="docs-Stack docs-Stack--stretch">
+      <div class="SplitButton">
+        <button type="button" class="Button Button--large Button--primary">
+          <span class="accessibility-hidden">Check</span>
+          <svg width="24" height="24" aria-hidden="true" class="mr-400">
+            <use xlink:href="/assets/icons/svg/sprite.svg#check-plain" />
+          </svg>
+          Button
+        </button>
+        <div class="Dropdown">
+          <button
+            type="button"
+            class="Button Button--large Button--primary"
+            data-spirit-toggle="dropdown"
+            data-spirit-target="#dropdown-primary-1"
+            aria-expanded="false"
+            aria-controls="dropdown-primary-1"
+            data-spirit-autoclose="true"
+          >
+            More
+            <svg width="24" height="24" aria-hidden="true" class="ml-400">
+              <use xlink:href="/assets/icons/svg/sprite.svg#chevron-down" />
+            </svg>
+          </button>
+          <div class="DropdownPopover" data-spirit-placement="bottom-end" id="dropdown-primary-1">
```

Comment:
I would consider avoiding numbers in IDs. From my experience, you will end up in the future with the need to add an example between `-1` and `-2`, and you will end up with renumbering everything. :)